sklearn/random_projection.py

```diff
@@ -402,8 +402,7 @@ def transform(self, X):
 
         Returns
         -------
-        X_new : numpy array or scipy sparse of shape [n_samples, n_components]
-            Projected array.
+        X_new : Project numpy array or scipy.sparse of shape [n_samples, n_components].
```

Comment:
What comes after the : should only be a description of type. The former text was correct